app/api_alpha/validators.py

```diff
@@ -18,6 +18,11 @@ def __call__(self, value):
         if rnb_id is None and shape is None:
             raise serializers.ValidationError("Either rnb_id or shape is required.")
 
+        if rnb_id is not None and shape is not None:
+            raise serializers.ValidationError(
+                "You can't provide a rnb_id and a shape, you should remove the shape."
```

Comment:
Je ne sais pas si on doit vraiment dire à l'utilisateur ce qu'il doit faire ici, on ne sait pas vraiment dans quelle situation il sera. Expliquer l'erreur serait plus neutre. Qu'en penses-tu ?

```suggestion
                "You can't provide both a rnb_id and a shape."
```

Reply:
Je me suis dit que globalement, on préférait, à choisir, un RNB ID plutôt qu'une géométrie. Ca nous évitera plus tard de devoir faire des rapprochements, non ?